terminology.md

```diff
@@ -421,6 +421,34 @@ This means demonstrating both (1) that it addresses a significant developer need
 
 Additionally, in a shorter-term sense, it could overwhelm developers to have too many syntax-heavy features ship in too narrow of a timeframe.
 
+## In-Band
+
+### Definition
+
+Used mainly in opposition to [Out of band](#out-of-band), denoting that all configuration of a feature or a behavior is done within the context of ECMASCript source.
+
+### Example
+
+- The stage 3 Import assertions proposal assert pragma
```

Comment:
Another possible nice example:
- the `"use strict"` directive is an in-band way to define the "language mode"
- script vs module is defined by out-of-band configuration (`type="module"` in HTML, or `.mjs`/`"type":"module"` in Node.js)

Reply:
Thank you for additional examples!  Will add that in

Reply:
nit: `.mjs` / `.js` + `"type":"module"` in node.js